lib/main.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'routes/home.dart';
```

Comment:
nit: separate the block of `package:` imports from the block of relative imports by a blank line.